File: Day-32/SumOfTriplets.js
function SumOfTriplets(arr, target) {
  let returnArray = [];
  let arrLength = arr.length;
  for (let i = 0; i < arrLength; i++) {
    for (let j = i + 1; j < arrLength; j++) {
      for (let k = j + 1; k < arrLength; k++) {
        let sum = arr[i] + arr[j] + arr[k];
        let sumArray = [arr[i], arr[j], arr[k]];
        if (sum == target) {
          returnArray.push(sumArray);
          if (returnArray.length != 0) {
            return returnArray;
          } else {
            return -1;
          }
        }
      }
    }
  }
  return -1;
}

console.log(SumOfTriplets([1, 2, 3, 1, 2, 3], 6));

File: Day-32/SumOfTriplets.py
def findTriplets(arr, n, k):
    
    # Write your code here.
    returnArray = []
    for i in range(0, n):
        for j in range (i+1, n):
            for l in range(j+1, n):
                sum = arr[i]+arr[j]+arr[l]
                if(sum==k):
                    sumArray = [arr[i], arr[j], arr[l]]
                    returnArray.append(sumArray)
                    if(len(returnArray)!=0):
                        return returnArray
                    else:
                        return -1
    return returnArray

print(findTriplets([1, 2, 3 ,1 ,2 ,3], 6, 6))
